frontend/src/actions/users.ts

```diff
@@ -70,17 +70,20 @@ export const fetchUserWithFollows = (
   };
 };
 
-export const followUser = (
-  userId: number,
-  token: string,
-  callback: Function = () => {}
-) => {
+export const followUser = (data: {
+  user_id: number;
+  token: string;
+  callback: Function;
```

Comment:
this will result and error because `;` instead of `,`

Reply:
no error